docs/typography.md

```diff
@@ -2,4 +2,27 @@
 layout: default
 title: Typography | UW IT Design System
 ---
-## Typography
\ No newline at end of file
+
+## Typography
+
+Use the type scale to apply consistent typographic design.
+
+### Typefaces
+
+The UW Madison Design System uses exclusively `Roboto`. The type was chosen
+because it is flexible, accessible and free to use, which is necessary to
+accommodate projects across UW’s campus.
+
++ Primary Font: Roboto
++ Fallback Fonts: Helvetica, Arial
++ Title Font: Verlag
```

Comment:
I don't think we're recommending Verlag as part of this style guide

Reply:
That's a copy-and-paste from Megan's initial cut at [Typography Guidelines](https://docs.google.com/document/d/1YWe6gSS6wyjwmEXtoCb4IiL9Ami9YqBtW8T27-fBXPY/edit#heading=h.4yo42mb9132t); cf [Slack thread](https://itdesignsystem.slack.com/archives/CGK6V4909/p1551802679006100).